AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTests.m

```diff
@@ -363,23 +413,53 @@ - (void)testTrackEventWithInvalidName {
   OCMVerifyAll(analyticsMock);
 }
 
+- (void)testTrackEventWithTypedPropertiesNilAndInvalidName {
+
+  // If
+  NSString *invalidEventName = nil;
+  id analyticsMock = OCMPartialMock([MSAnalytics sharedInstance]);
+  [MSAppCenter configureWithAppSecret:kMSTestAppSecret];
+  id channelUnitMock = OCMProtocolMock(@protocol(MSChannelUnitProtocol));
+  id channelGroupMock = OCMProtocolMock(@protocol(MSChannelGroupProtocol));
+  OCMStub([channelGroupMock addChannelUnitWithConfiguration:OCMOCK_ANY]).andReturn(channelUnitMock);
+  [[MSAnalytics sharedInstance] startWithChannelGroup:channelGroupMock
+                                            appSecret:kMSTestAppSecret
+                              transmissionTargetToken:nil
+                                      fromApplication:YES];
+
+  // FIXME: logManager holds session tracker somehow and it causes other test failures. Stop it for hack.
```

Comment:
Just a side note:
a) we're not using `logManager` as a term anymore
b) we need to look into this again, it shouldn't be like that.

Reply:
I can remove that (it was just copied from another test)